### PR TITLE
Allows for Predefined Schemas

### PR DESCRIPTION
**Pull request by hhff**
- [ ] Had to move from self-hosted to github hosting to support instance parameters
- [ ] This means we use rawgit to host the dist directly here
- [ ] Also means that in the future we'll be able allow users to install specific versions and not receive updates if they want a specific version

### CODE REVIEW

File: app/services/extension.js [hunk 1]
```diff
@@ -57,10 +63,48 @@ export default Service.extend({
...
+    if (!shorthand.length) return;
+
+    const parsedSchemaFields = 
+      shorthand.split(",").map(tuple => tuple.split(":").map(t => t.trim()));
+    const existingSchema = get(this, 'data._schema') || [];
+
+    const loadedSchema = parsedSchemaFields.reduce((acc, fieldTuple) => {
+      const [key, type] = fieldTuple;
+      if (key.length === 0) {
+        console.warn(
```
Comment (aliciakw) [inline app/services/extension.js]: these are throwing errors in the tests... if you want to keep them, mb add an eslint-ignore here, ando n line 93?

File: app/services/extension.js [hunk 2]
```diff
@@ -57,10 +63,48 @@ export default Service.extend({
...
+          `Contentful Fragment: Your Predefined Schema included a blank schema key! Please refer to the documentation.`
+        );
+        return acc;
+      }
+      if (!TYPES.includes(type)) {
+        console.warn(
+          `Contentful Fragment: Your Predefined Schema included unknown type: ${type}. Must be one of <${TYPES.join(' ')}>`
+        );
+        return acc;
+      };
```
Comment (aliciakw) [inline app/services/extension.js]: unnecessary semi colon ;)

**Review by aliciakw — APPROVED** [review]
this is rad, ran it locally!

**Comment by hhff** [comment]
@aliciakw updated!

**Pull request merged**